front/lib/api/assistant/configuration.ts

```diff
@@ -346,6 +517,7 @@ export async function createAgentConfiguration(
     sId: agentConfig.sId,
     version: agentConfig.version,
     scope: agentConfig.scope,
+    relationOverride: formerAgentRelationOverride,
```

Comment:
Let's just go for `null` here this is fine. a relationOverride is relative to a user. We can make the argument that an assistant creation is not a user specific thing so it can be expected by convention that relationOveride is null always here no?

Let's kill the diff in that function with :fire:

Reply:
Since we have to grab former agent anyways (for scope integrity check, done here rather than in DB) the code diff will be minor, and we'll have to be wary of the convention (if the function is ever reused elsewhere, etc.) VS having the code do the work for us

Not strongly held => if you still disagree, i'll kill the diff without further ado

Reply:
OK